scrapers/DollsHub.yml

```
@@ -0,0 +1,41 @@
+name: DollsHub
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - filthygapers.com
+      - gynoexclusive.com
+      - ihuntmycunt.com
+      - maturegapers.com
+      - maturegynoexam.com
+      - maturegynospy.com
+      - nastypublicsex.com
+      - oldsfuckdolls.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="col-sm-12 col-lg-12 col-md-12 scence-title-name"]/h2
+      Performers:
+        Name: //li/span/a
+      Details:
+        selector: //h4[contains(text(),"Description")]/following-sibling::text()
```

The text selector get some more comments as results (not included in stash though)
Getting the `<p>` instead returns only a single result
```suggestion
        selector: //h4[contains(text(),"Description")]/following-sibling::p
```